neptune/__init__.py

```diff
@@ -136,27 +136,22 @@ def init(project_qualified_name=None, api_token=None, proxies=None, backend=None
 
     if project_qualified_name is None:
```

Comment:
Right now we're not accepting string which is `None`, but empty string is ok.
Does empty string make sense? If we'd force non empty string, the code could be simplified.